File: A 486 - Calculating Function/main.py
# A 486 - Calculating Function - Codeforces
# Link : https://codeforces.com/contest/486/problem/A

x = int(input())
if (x % 2 == 0):
    print(int(x / 2))
else:
    print(int(-1 * (x + 1) / 2))

File: A 486 - Calculating Function/main.js
// Calculating Function

// Link : https://codeforces.com/contest/486/problem/A

const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input;

readline.on("line", (line) => (input = parseInt(line)));
readline.on("close", () => {
  if (input % 2 == 0) {
    console.log(Math.floor(input / 2));
  } else {
    console.log(Math.floor((-1 * (input + 1)) / 2));
  }
});